src/lib/middleware/sharifyLocals.ts

```diff
@@ -0,0 +1,20 @@
+import type { NextFunction } from "express"
+import type { ArtsyRequest, ArtsyResponse } from "./artsyExpress"
+
+import { parse } from "url"
+
+/**
+ * Contains the subset of sharify locals that are required for all force routes.
+ */
+export function sharifyLocalsMiddleware(
+  req: ArtsyRequest,
+  res: ArtsyResponse,
+  next: NextFunction
+) {
+  const ua = req.get("user-agent") || ""
+
+  res.locals.sd.CURRENT_PATH = parse(req.url).pathname
+  res.locals.sd.EIGEN = ua.match("Artsy-Mobile") != null
```

Comment:
The `REFLECTION` key seems almost identical to `EIGEN`, if we are breaking this out I think it is worth adding here as well.

Reply:
Since no modern pages depend on Reflection (🤞 hopefully), we might get away without adding this to the novo template.